source/index.rst

```diff
@@ -57,7 +58,8 @@ Phoenix Pro currently offers the following features and will further expand:
 
       Documentation for device specific configuration, troubleshooting and setup instructions.
 
-.. grid:: 3
+.. grid:: 1 2 3 3
+   :gutter: 3
```

Comment:
Should this grid be combined with the grid above? Right now, "medium" screens have a weird wrapping behavior where you get rows of: 2, 1, 2, 1.

Reply:
Yes. Resolved.